apps/api/v2/src/modules/organizations/repositories/organizations-event-types.repository.ts

```diff
@@ -6,52 +7,52 @@ import { Injectable } from "@nestjs/common";
 export class OrganizationsEventTypesRepository {
   constructor(private readonly dbRead: PrismaReadService, private readonly dbWrite: PrismaWriteService) {}
 
-  async getTeamEventType(teamId: number, eventTypeId: number) {
+  async getTeamEventType(teamId: number, eventTypeId: number): Promise<DatabaseTeamEventType | null> {
     return this.dbRead.prisma.eventType.findUnique({
       where: {
         id: eventTypeId,
         teamId,
       },
-      include: { users: true, schedule: true, hosts: true },
+      include: { users: true, schedule: true, hosts: true, destinationCalendar: true },
     });
   }
 
-  async getTeamEventTypeBySlug(teamId: number, eventTypeSlug: string) {
+  async getTeamEventTypeBySlug(teamId: number, eventTypeSlug: string): Promise<DatabaseTeamEventType | null> {
     return this.dbRead.prisma.eventType.findUnique({
       where: {
         teamId_slug: {
           teamId,
           slug: eventTypeSlug,
         },
       },
-      include: { users: true, schedule: true, hosts: true },
+      include: { users: true, schedule: true, hosts: true, destinationCalendar: true },
     });
   }
 
-  async getTeamEventTypes(teamId: number) {
+  async getTeamEventTypes(teamId: number): Promise<DatabaseTeamEventType[]> {
     return this.dbRead.prisma.eventType.findMany({
       where: {
         teamId,
       },
-      include: { users: true, schedule: true, hosts: true },
+      include: { users: true, schedule: true, hosts: true, destinationCalendar: true },
     });
   }
 
-  async getEventTypeById(eventTypeId: number) {
+  async getEventTypeById(eventTypeId: number): Promise<DatabaseTeamEventType | null> {
```

Comment:
Why do we need to explicitly typecast prisma calls? Response should be inferred and work automatically.

Reply:
https://github.com/calcom/cal.com/pull/16314/commits/0612d0554290adcd285defa0b07cea18bbad2949